manifests/init.pp

```diff
@@ -136,10 +135,9 @@
 #   Also see [`makestep_seconds`](#makestep_seconds).
 # @param queryhosts
 #   This adds the networks, hosts that are allowed to query the daemon.
-#   Note that `port` needs to be set for this to work.
 # @param port
-#   Port the service should listen on, to be used in combination with `queryhosts`.
-#   Module default is `0` to prevent accidental activation of server mode.
+#   Port the service should listen on.
+#   Module default is undef.
```

Comment:
what does a default of `undef` mean?
why is it significant that it's a "module default"? as opposed to?
why do you have to spell it out here, rather than that default being pulled out from the code?

Reply:
Good questions

Is this better?

```
#   Port the service should listen on. Module default is `undef` which means that port
#   isn't added to chrony.conf, and chrony listens to the default ntp port 123 if
#   `queryhosts` is used.
```

Reply:
much better.!